tests/testthat/test-dplyr.R

```diff
@@ -535,6 +537,8 @@ test_that("result from dplyr::compute() has remote name", {
 })
 
 test_that("dplyr::summarize() emits an error for summarizer using one-sided formula", {
+  skip_if_not(dbplyr_uses_ops())
```

Comment:
This works now because it uses dbplyr. Is there any good reason why this should not work?